src/Api/Model/Languageforge/Lexicon/Config/LexConfig.php

```diff
@@ -127,14 +127,14 @@ public static function flexOptionlistCode($fieldName)
      * @var array
      */
     private static $flexOptionlistNames = [
-        "grammatical-info" => "Part of Speech",
+        "grammatical-info" => "Grammatical Category",
         "semantic-domain-ddp4" => "Semantic Domain",
         "domain-type" => "Academic Domains",
         self::ENVIRONMENTS => "Environments",
         "location" => "Location",
         "usage-type" => "Usages",
         "reversal-type" => "Reversal Entries",
-        "sense-type" => "Type",
+        "sense-type" => "Sense Type",
```

Comment:
I would like you to double check that the label in FLEx is actually "Sense Type".  It already falls under the "Sense" heading, so to me the label "Sense Type" is redundant.  However, if Verna could not easily determine what the field was, maybe that's an argument for "Sense Type".

I am not on Windows, so cannot easily check.  I knew that eventually you will need to be able to fire up FLEx (on Windows presumably) so now's a good time to figure out how to do that :). FLEx does run on Linux, although the setup is harder (it's more than a file download like a windows installer is)

Reply:
"Sense Type" is a Sense field in FLEx
![image](https://user-images.githubusercontent.com/12587509/221171860-e57848b6-c2dd-449d-b69a-4b41766e65c9.png)

Reply:
@myieye to the rescue...now @longrunningprocess can put off installing Windows for another day 😆

Reply:
thank you more than words can convey @myieye  😆

Reply:
@megahirt I actually somehow didn't notice your second paragraph about @longrunningprocess installing FLEx 😆. W00ps 😁.